src/ogma/features/nodeGrouping.ts

```diff
@@ -173,7 +175,13 @@ export class NodeGroupingTransformation {
         },
         color: (node: Node | undefined) => {
           if (node !== undefined) {
-            return this._ogma.LKStyles.nodeAttributes.color(node.getData());
+            return this._ogma.LKStyles.nodeAttributes.color({
```

Comment:
can you explain what happens here? Why pass an object to 'color'?

Reply:
It is to get the color of the categories of the sub nodes.
color expects an itemData, as virtual nodes don't have the same data as the normal node I am building it.